platform/security/security-migratable/src/main/java/org/codice/ddf/security/migratable/impl/SecurityMigratable.java

```diff
@@ -97,6 +99,9 @@ public void doExport(ExportMigrationContext context) {
         .forEach(ExportMigrationEntry::store);
     LOGGER.debug("Exporting PDP files from [{}]...", SecurityMigratable.PDP_POLICIES_DIR);
     context.getEntry(SecurityMigratable.PDP_POLICIES_DIR).store();
+    LOGGER.debug(
```

Comment:
✏️ Is this something we should be auditing in the security log?

Reply:
The migration implementation is already auditing all files and directories being migrated.